org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "opent2t-translator-com-wink-sensorpod",
+    "version": "0.2.2",
```

Comment:
Should be 0.2.0

---
In reply to: [105221495](https://github.com/openT2T/translators/pull/203#discussion_r105221495) [](ancestors = 105221495)